pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -19,6 +19,7 @@ package predicates
 import (
 	"context"
 	"fmt"
+	"k8s.io/kubernetes/pkg/scheduler/internal/podinfo"
```

Comment:
Please move this import down to the other `k8s.io` imports. The preferred style is usually,
```
<standard library imports>
<blank line>
<everything else>
```

When I'm writing imports, I like to add one extra level of organization:
```
<standard library imports>
<blank line>
<libraries not part of scheduler>
<blank line>
<scheduler libraries>
```

Reply:
(and please make the same fix in the other files as well.)

Reply:
This happened as a result of auto refactoring by GoLand. I have squashed and rebased the commits. Please review the same.

Reply:
Looks like the import is still in the top section. Can you override GoLand's placement? (if you manually move it, does GoLand put it back?)